code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm

```diff
@@ -0,0 +1,285 @@
+
+/datum/reagent/medicine/C2/convermol/overdose_process(mob/living/carbon/human/M)
+	metabolization_rate += 1
+	..()
+	return TRUE
+
+/datum/reagent/medicine/C2/tirimol
+	name = "Tirimol"
+	description = "An oxygen deprivation medication that causes fatigue. Prolonged exposure causes the patient to fall asleep once the medicine metabolizes."
+	color = "#FF6464"
+	var/drowsycd = 0
+
+/datum/reagent/medicine/C2/tirimol/on_mob_life(mob/living/carbon/human/M)
+	M.adjustOxyLoss(-1)
+	M.adjustStaminaLoss(2)
+	if(drowsycd && (world.time > drowsycd))
+		M.drowsyness += 10
+		drowsycd = world.time + (45 SECONDS)
+	else if(!drowsycd)
+		drowsycd = world.time + (15 SECONDS)
+	..()
+	return TRUE
+
+/datum/reagent/medicine/C2/tirimol/on_mob_end_metabolize(mob/living/L)
+	if(current_cycle > 20)
+		L.Sleeping(200)
+	..()
+
+/******TOXIN******/
+/*Suffix: -iver*/
+
+#define FIZ_WEAKRATE	0.005
+
+/datum/reagent/medicine/C2/fiziver //fiz = phys ok?
+  name = "Fiziver"
+  description = "An antitoxin that temporarily weakens the user, making them susceptible to other forms of damage. Weakness and toxin healing scales with exposure."
+  overdose_threshold = 11 // Scales every damage so naturally quite abusable. Luckily the OD purges the chem which resets the damage mod *wink*
+  var/reset_mods = 0 //amount, not bool
+
+/datum/reagent/medicine/C2/fiziver/on_mob_life(mob/living/carbon/human/M)
+	var/maths = FIZ_WEAKRATE * current_cycle
+	reset_mods += maths
+	var/datum/physiology/phis = M.physiology
+	phis.brute_mod += maths
+	phis.burn_mod += maths
+	phis.oxy_mod += maths
+	phis.stamina_mod += maths
+	M.adjustToxLoss(round(reset_mods*-1000,0.01)) //Math is fun!
+	..()
+	return TRUE
+
+/datum/reagent/medicine/C2/fiziver/on_mob_delete(mob/living/carbon/human/M)
+	var/datum/physiology/phis = M.physiology
+	phis.brute_mod = max(phis.brute_mod - reset_mods, 1)
+	phis.burn_mod = max(phis.burn_mod - reset_mods, 1)
+	phis.oxy_mod = max(phis.oxy_mod - reset_mods, 1)
+	return ..()
+
+/datum/reagent/medicine/C2/fiziver/overdose_process(mob/living/carbon/human/M)
+	var/maths = reset_mods*3
+	M.adjustBruteLoss(maths)
+	M.adjustFireLoss(maths)
+	M.adjustOxyLoss(maths)
+	..()
+	M.reagents.del_reagent(type)
+	return TRUE
+
+#undef FIZ_WEAKRATE
+
+/datum/reagent/medicine/C2/multiver //made up chem, it's a PALLETTE cleanser hehe
+	name = "Multiver"
+	description = "An antitoxin that scales with the more unique medicines in the body as well as purges chems (including itself). Causes lung damage."
+
+/datum/reagent/medicine/C2/multiver/on_mob_life(mob/living/carbon/human/M)
+	var/medibonus = 0 //it will always have itself which makes it REALLY start @ 1
+	for(var/r in M.reagents.reagent_list)
+		var/datum/reagent/the_reagent = r
+		if(istype(the_reagent, /datum/reagent/medicine))
+			medibonus += 1
+	M.adjustToxLoss(0.5 * medibonus)
+	for(var/datum/reagent/R in M.reagents.reagent_list)
+		M.reagents.remove_reagent(R.type, medibonus*0.5)
+		M.adjustOrganLoss(ORGAN_SLOT_LUNGS, medibonus)
```

Comment:
this part feels like a really bad idea.

Reply:
I think there is a deathmix that can deal 110 lung damage in one tick with less than 2.5u mix if I am reading the code correctly.

This type scaling mechanics breed problems.

Reply:
indentation issue

Reply:
?

Reply:
It's suppose to not be in the loop.